qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -508,6 +508,7 @@ def gen_fc_sched(*args, **kwargs):
                 _phase = phase_expr(*args, **kwargs)
                 return instructions.SetPhase(_phase, channel) << t0
 
+            # ToDo(4872) remove the use of deprecated constructor Schedule(*schedulers)
```

Comment:
So, I noticed these are not covered in tests (so it wasn't failing/generate warnings).
I still have to find the best way to replace this, I couldn't manage to have e a proper look at `ParameterizedSchedule` today, but at first glance it seems we might need a new method to insert a "callable"?

Reply:
I don't mind leaving the ParameterizedSchedule as is, it will be deprecated by #5073.